### PR TITLE
chore!: Upgrade media chrome to 4.1.0

### PR DESCRIPTION
**Pull request by AdamJaggard**
~~Draft PR until both current and classic themes are updated with new menu elements.~~

ts-ignore was required for the theme imports from player.style, build works fine like this but TS complains of missing modules without the ignore.

Removed `selectmenu` menus and used new menus in current and classic theme

This change also brings in tooltips and they are enabled in the default theme (open to styling suggestions), new `no-tooltips` attribute / `noTooltips` prop available available also to disable them regardless of which theme is enabled.

image

I also updated the standard player example in the vanilla examples to use a new video that has captions and multiple audio tracks for easier testing.

### CODE REVIEW

**Comment by cjpillsbury** [comment]
@AdamJaggard haven't reviewed for this yet in your PR, but make sure you catch any of the updates @luwes did in #961 for TS migration.

File: packages/mux-player/src/themes/classic/classic.html [hunk 1]
```diff
@@ -18,17 +17,23 @@
...
       color: var(--_primary-color);
       display: inline-block;
       width: 100%;
       height: 100%;
     }
 
+    media-rendition-menu,
+    media-audio-track-menu,
+    media-captions-menu {
+      position: absolute; /* ensure they don't take up space in DOM on load */
```
Comment (luwes) [inline packages/mux-player/src/themes/classic/classic.html:34]: 👍

File: packages/mux-player/src/themes/classic/classic.html [hunk 2]
```diff
@@ -336,44 +341,60 @@
...
+      </svg>
+    </media-captions-menu-button>
+    <media-captions-menu
+      hidden
+      anchor="auto"
+      part="bottom captions menu"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+      exportparts="menu-item"
+      part="bottom captions menu"
```
Comment (luwes) [inline packages/mux-player/src/themes/classic/classic.html:396]: duplicate part attribute here

File: packages/mux-player/src/themes/gerwig/gerwig.html [hunk 3]
```diff
@@ -641,55 +647,56 @@
...
+      </svg>
+    </media-captions-menu-button>
+    <media-captions-menu
+      hidden
+      anchor="auto"
+      part="bottom captions menu"
+      disabled="{{disabled}}"
+      aria-disabled="{{disabled}}"
+      exportparts="menu-item"
+      part="bottom captions menu"
```
Comment (luwes) [inline packages/mux-player/src/themes/gerwig/gerwig.html:676]: here too

File: packages/mux-player/src/themes/gerwig/gerwig.html [hunk 4]
```diff
@@ -641,55 +647,56 @@
...
-          />
-          <path
-            d="M12.6003 10.258C11.9763 10.258 11.4382 10.1218 10.9851 9.8488C10.532 9.5758 10.1824 9.1945 9.93685 8.7057C9.69075 8.2164 9.56775 7.651 9.56775 7.0089C9.56775 6.361 9.69075 5.7926 9.93685 5.3033C10.1824 4.8145 10.532 4.4322 10.9851 4.1558C11.4382 3.8799 11.9763 3.7417 12.6003 3.7417C13.3386 3.7417 13.9416 3.9219 14.4094 4.2817C14.8777 4.6421 15.1745 5.146 15.3005 5.7939H14.2385C14.1545 5.4282 13.9743 5.1367 13.6985 4.9209C13.4227 4.7051 13.0535 4.5967 12.5916 4.5967C11.9617 4.5967 11.4637 4.8115 11.0975 5.2403C10.7318 5.6695 10.5487 6.2589 10.5487 7.0089C10.5487 7.753 10.7318 8.338 11.0975 8.7638C11.4637 9.1901 11.9618 9.403 12.5916 9.403C13.0535 9.403 13.4227 9.3009 13.6985 9.0968C13.9744 8.8932 14.1546 8.6139 14.2385 8.2599H15.3005C15.1745 8.8839 14.8776 9.3732 14.4094 9.7272C13.9416 10.0812 13.3386 10.258 12.6003 10.258Z"
-          />
+  <template partial="CaptionsMenu">
+    <media-captions-menu-button>
```
Comment (luwes) [inline packages/mux-player/src/themes/gerwig/gerwig.html:651]: no part attribute here

File: packages/mux-player/src/themes/classic/classic.html [hunk 5]
```diff
@@ -336,44 +341,60 @@
...
-        </svg>
-      </media-captions-button>
-      <media-captions-listbox slot="listbox" exportparts="option" part="bottom captions listbox"></media-captions-listbox>
-    </media-captions-selectmenu>
+    <media-captions-menu-button>
```
Comment (luwes) [inline packages/mux-player/src/themes/classic/classic.html:381]: no part attribute here

**Comment by AdamJaggard** [comment]
@cjpillsbury did some more testing with your changes in and it's looking good on my end

File: packages/mux-player/package.json [hunk 6]
```diff
@@ -96,24 +96,25 @@
   "dependencies": {
     "@mux/mux-video": "0.20.2",
     "@mux/playback-core": "0.25.2",
-    "media-chrome": "~3.2.5"
+    "media-chrome": "~4.1.0",
+    "player.style": "^0.0.7"
```
Comment (luwes) [inline packages/mux-player/package.json:100]: try upgrading to 0.0.8 for fixing the dup node_modules issue

Comment (luwes) [inline packages/mux-player/package.json:100]:
```suggestion
    "player.style": "^0.0.8"
```

Comment (AdamJaggard) [reply packages/mux-player/package.json]: Done! see commit below for affect

**Review by luwes — APPROVED** [review]
LGTM! 👍

**Pull request merged**